definitions/npm/react-storybook_v1.x.x/flow_>=v0.30.x/react-storybook_v1.x.x.js

```diff
@@ -0,0 +1,8 @@
+declare module '@kadira/storybook' {
+  declare interface Story {
+    add: (storyName: string, callback: Function) => Story;
+  }
+
+  declare function storiesOf(name: string, module: any): Story;
+  declare function action(name: string): Function;
+}
```

Comment:
Can you add the entire API interface? Looks like there is all of this: https://github.com/kadirahq/react-storybook/blob/master/src/client/index.js#L3-L11

Also, a couple things that would make this better:

- Specify the callbacks more than just `Function`, i.e. `(..., callback: (arg: boolean) => string) => Story`
- Specify what the `any`'s are. Either the specific type or use `mixed`.